schemas/com.snowplowanalytics.snowplow/button_click/jsonschema/1-0-0

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a button click event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "button_click",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "action": {
+      "type": "string",
+      "description": "The action associated with the button click",
+      "minLength": 1,
+      "maxLength": 4096
+    },
+    "elementId": {
+      "description": "The ID of the element clicked if present",
+      "type": [
+        "string",
+        "null"
+      ],
+      "minLength": 1,
+      "maxLength": 4096
+    },
+    "elementClasses": {
+      "description": "An array of class names from the element clicked",
+      "type": [
+        "array",
+        "null"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "elementName": {
+      "description": "The name of the element if present",
```

Comment:
Would this be the value of a `name` property? I don't think it's very common to set the `name` for buttons, but might be wrong.

Reply:
I got `name` this from the [button spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#name), but yeah, I'm not sure how common it would actually be to set it (and we want to avoid overlapping this feature with form tracking). Perhaps it can be removed?

Reply:
Nevermind, I have seen this in other button click events as well, happy to leave as is.